web_src/css/repo/home.css

```diff
@@ -24,7 +24,7 @@
   border-top: 1px solid var(--color-secondary); /* same to .flex-list > .flex-item + .flex-item */
 }
 
-@media (max-width: 767.98px) {
+@media (max-width: 872px) { /* value is exactly where the file table starts to overflow */
```

Comment:
In my case, at least around 911px is necessary.
monitor is 1080p, OS is windows, browser is Chrome latest version

![image](https://github.com/user-attachments/assets/c15c9f97-b7ef-49ed-b697-bdc6c002eaf5)
![image](https://github.com/user-attachments/assets/a26102c7-4fa0-4c99-a886-78e07f097942)

Reply:
I would never believe such pixel-level fine-tuning could be right ......

Reply:
Actually this fix is a pixel-level fine-tuning.
And I think the table's width is not correctly calculated, so simply adjust `max-width` is not correct.

e.g. If there's only `Description`, you will see:
![image](https://github.com/user-attachments/assets/675f725a-5025-4a96-8963-1dfae57cecad)